src/main/java/com/easyrest/pages/OwnerPanel.java

```diff
@@ -0,0 +1,315 @@
+
+    private final By deleteWaiterButton = By.xpath("//li[contains (@class, 'MuiListItem-root')][1]//*[@viewBox='0 0 24 24']");
+    private final By administratorsMenuButton = By.xpath("//span[text() = 'Administrators']");
+    private final By addAdministratorButton = By.xpath("//button[@title = 'Add Administrator']");
+    private final By deleteAdministratorButton = By.xpath("//li[contains (@class, 'MuiListItem-root')][1]//*[@viewBox='0 0 24 24']");
+
+    public OwnerPanel(WebDriver driver) {
+        super(driver);
+    }
+
+    public OwnerPanel clickOnMenuItemUser() {
+
+        driver.findElement(personalInfoMenuButton).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnMenuButtonCurrentOrder() {
+
+        driver.findElement(currentOrderMenuButton).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnMenuButtonOrderHistory() {
+
+        driver.findElement(orderHistoryMenuButton).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnMenuButtonMyRestaurants() {
+
+        driver.findElement(myRestaurantsMenuButton).click();
+        return new OwnerPanel(driver);
+    }
+
+
+    public OwnerPanel clickOnButtonAddRestaurant() {
+
+        driver.findElement(addRestaurantButton).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnButtonOptionOfRestaurants() {
+
+        driver.findElement(optionsOfRestaurantButton).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnMenuItemArchiveRest() {
+
+        driver.findElement(archiveRestMenuItem).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnMenuItemUnarchiveRest() {
+
+        driver.findElement(unarchiveRestMenuItem).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnMenuItemManageRest() {
+
+        driver.findElement(manageRestMenuItem).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnButtonEditInfoRest() {
+
+        driver.findElement(editInfoRestButton).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnButtonUpdateInfoRest() {
+
+        driver.findElement(updateInfoRestButton).click();
+        return new OwnerPanel(driver);
+    }
+
+
+    public OwnerPanel clickOnMenuButtonMenues() {
+
+        driver.findElement(menuesMenuButton).click();
+        return new OwnerPanel(driver);
+    }
+
+
+    public OwnerPanel clickOnButtonWaitersMenu() {
+
+        driver.findElement(waitersMenuButton).click();
+        return new OwnerPanel(driver);
+    }
+
+    public OwnerPanel clickOnButtonAddWaiter() {
+
+        driver.findElement(addWaiterButton).click();
+        return new OwnerPanel(driver);
+    }
+
+
+    public OwnerPanel sendToFieldName(String name) {
+        driver.findElement(nameField).clear();
+        driver.findElement(nameField).sendKeys(name);
```

Comment:
```
driver.findElement(nameField).clear();
driver.findElement(nameField).sendKeys(name);
```

This is a duplicated logic, same as sendToFieldMail(String email), sendToFieldPassword(String password), sendToFieldPhone(String phone), etc.